lib/manifest.js

```diff
@@ -338,6 +338,18 @@ internals.manifest = [
         after: ['plugins'],
         example: { $requires: ['schmervice'], $value: internals.SchmerviceExample }
     },
+    {
+        place: 'subscriptions',
+        method: 'subscription',
+        signature: ['path', '[options]'],
+        list: true,
+        after: ['plugins'],
+        example: {
+            $required: ['nes'],
```

Comment:
This option is all about output of the file when it is created by hpal.  You only need the `$requires` field if you want to require nes at the top of the subscription file when using `hpal make`.
```suggestion
```

Reply:
Wasn't sure about that one! i'll make the changes